db/movie_info.csv

```diff
@@ -39,62 +39,62 @@ IT,1989,Horror,0,tt1396484,https://m.media-amazon.com/images/M/MV5BZDVkZmI0YzAtN
+Nosferatu,1922,Fantasy,Horror,1,tt0013442,https://m.media-amazon.com/images/M/MV5BMTAxYjEyMTctZTg3Ni00MGZmLWIxMmMtOGM2NTFiY2U3MmExXkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_SX300.jpg
+Paranorman,2012,Animation,Adventure,1,tt1623288,https://m.media-amazon.com/images/M/MV5BMjA1OTU1NDM3N15BMl5BanBnXkFtZTcwMjYxNTg0Nw@@._V1_SX300.jpg
+Pet Sematary,1989,Fantasy,Horror,0,tt0098084,https://m.media-amazon.com/images/M/MV5BYjBlNTBhYWQtMzg5Yi00NDA2LWJmMjYtZmM0ODhiYzkwYmY5XkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_SX300.jpg
+Pit and the Pendulum,1961,Drama,Horror,1,tt0055304,https://m.media-amazon.com/images/M/MV5BZGY1ODM0Y2EtYTBlNy00ZTQ5LWI5MzQtZTY2NzllYTM4ZTRlXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_SX300.jpg
+Poltergeist,2015,Horror,Thriller,0,tt0084516,https://m.media-amazon.com/images/M/MV5BNzliZmRlYTctYmNkYS00NzE5LWI1OWQtMTRiODY5MDMwMTVkXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_SX300.jpg
+Psycho,1960,Horror,Mystery,1,tt0054215,https://m.media-amazon.com/images/M/MV5BNTQwNDM1YzItNDAxZC00NWY2LTk0M2UtNDIwNWI5OGUyNWUxXkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_SX300.jpg
+Pumpkinhead,1988,Fantasy,Horror,1,tt0095925,https://m.media-amazon.com/images/M/MV5BMTQ0YTQ2ZTUtNGFkOC00YzI2LTkwNDktN2Y0YjI4NGVjMDM3L2ltYWdlXkEyXkFqcGdeQXVyNTAyODkwOQ@@._V1_SX300.jpg
+Re-Animator,1985,Comedy,Horror,0,tt0089885,https://m.media-amazon.com/images/M/MV5BNDRjNjRlNmYtZGVkOS00ODA1LWFmMTEtZmY1MWY5ZGM5OWUxXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_SX300.jpg
+Rosemary's Baby,1968,Drama,Horror,0,tt0063522,https://m.media-amazon.com/images/M/MV5BZmEwZGU2NzctYzlmNi00MGJkLWE3N2MtYjBlN2ZhMGJkZTZiXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_SX300.jpg
+Saw,2004,Horror,Mystery,0,tt0387564,https://m.media-amazon.com/images/M/MV5BMzQ2ZTBhNmEtZDBmYi00ODU0LTgzZmQtNmMxM2M4NzM1ZjE4XkEyXkFqcGdeQXVyNjE5MjUyOTM@._V1_SX300.jpg
+Scary Stories to Tell in the Dark,2019,Adventure,Horror,1,tt3387520,https://m.media-amazon.com/images/M/MV5BMmI1MGE0ODMtYWRlZC00ZDUxLWIyNGItYjgyNzhhMTRlOTI2XkEyXkFqcGdeQXVyOTQ5MTIwMjM@._V1_SX300.jpg
+Scooby-Doo,2002,Adventure,Comedy,1,tt0267913,https://m.media-amazon.com/images/M/MV5BMTg4MzMzMTY0OF5BMl5BanBnXkFtZTYwNzM3MTg2._V1_SX300.jpg
+Scream,	1996,Horror,Mystery,1,tt0117571,https://m.media-amazon.com/images/M/MV5BMjA2NjU5MTg5OF5BMl5BanBnXkFtZTgwOTkyMzQxMDE@._V1_SX300.jpg
+Silent Hill,2006,Horror,0,tt0384537,https://m.media-amazon.com/images/M/MV5BOWQzNTU3NGUtNDAxMi00ZjFjLTkyYWUtYjgwZDJiMDljZWY4XkEyXkFqcGdeQXVyNTAyODkwOQ@@._V1_SX300.jpg
+Sleepy Hollow,1999,Fantasy,Horror,0,tt0162661,https://m.media-amazon.com/images/M/MV5BMTQ4Yjk5ODEtNGFkOS00OTY1LTgwYTYtOTFkNmU2NzBmNGM2XkEyXkFqcGdeQXVyNDQ2MTMzODA@._V1_SX300.jpg
+Sleepy Hollow,1999,Fantasy,Horror,1,tt0162661,https://m.media-amazon.com/images/M/MV5BMTQ4Yjk5ODEtNGFkOS00OTY1LTgwYTYtOTFkNmU2NzBmNGM2XkEyXkFqcGdeQXVyNDQ2MTMzODA@._V1_SX300.jpg
+Something Wicked This Way Comes,1983,Fantasy,NULL,0,tt0086336,https://m.media-amazon.com/images/M/MV5BZjAwNWM2ZjItMTgwYi00YjMzLTkyZjEtYTU2YjFiODkzMjA5XkEyXkFqcGdeQXVyNTAyODkwOQ@@._V1_SX300.jpg
```

Comment:
Is the "NULL" written by you, or is that how a null field appears in the csv? Just concerned that the DB will record a genre as string "null" instead of just giving it a null value.